kafka-persistence/src/main/java/com/netflix/conductor/dao/es5/index/ElasticSearchRestKafkaDAOV5.java

```diff
@@ -0,0 +1,73 @@
+package com.netflix.conductor.dao.es5.index;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.netflix.conductor.annotations.Trace;
+import com.netflix.conductor.common.metadata.events.EventExecution;
+import com.netflix.conductor.common.metadata.tasks.Task;
+import com.netflix.conductor.common.metadata.tasks.TaskExecLog;
+import com.netflix.conductor.common.run.TaskSummary;
+import com.netflix.conductor.common.run.Workflow;
+import com.netflix.conductor.common.run.WorkflowSummary;
+import com.netflix.conductor.core.events.queue.Message;
+import com.netflix.conductor.dao.ProducerDAO;
+import com.netflix.conductor.dao.kafka.index.utils.RecordTypeConstants;
+import com.netflix.conductor.elasticsearch.ElasticSearchConfiguration;
+import org.elasticsearch.client.RestClient;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+@Trace
+@Singleton
+public class ElasticSearchRestKafkaDAOV5 extends ElasticSearchRestDAOV5 {
```

Comment:
Hi @apanicker-nflx This class extends ElasticSearchRestDAOV5 so that all read operations will route to ElasticSearchRestDAOV5. But overrides all index operations which publish to kafka topic.